File: Nejman.Newtone.Core/Models/TrackModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Nejman.Newtone.Core.Models
{
    public class TrackModel
    {
        public string Artist { get; }
        public string Title { get; }
        public string Path { get; }
        internal TrackModel(string artist, string title, string path)
        {
            Artist = artist;
            Title = title;
            Path = path;
        }
    }
}

File: app/src/main/java/com/nejman/nsec/music_player/core/models/TrackModel.java
package com.nejman.nsec.music_player.core.models;

import android.widget.ImageButton;
import android.widget.ImageView;

public class TrackModel {
    public String artist;
    public String title;
    public String path;

    public TrackModel(String artist, String title, String path)
    {
        this.artist = artist;
        this.title = title;
        this.path = path;
    }
}
